golem-worker-executor-base/src/durable_host/blobstore/container.rs

```diff
@@ -35,6 +35,7 @@ impl<Ctx: WorkerCtx> HostContainer for DurableWorkerCtx<Ctx> {
         &mut self,
         container: Resource<Container>,
     ) -> anyhow::Result<Result<String, Error>> {
+        let _permit = self.begin_async_host_function().await?;
```

Comment:
Why the introduction of this change? It increases the boilerplate required, and also will be difficult to enforce, making it error-prone. Finally, if we did need such a change, we would want to make `record_host_function_call` always return the permit, since that must be the first thing done in these functions, anyway.

Reply:
The primary reason was my OpenAPI3 implementation testing. My tests couldn't handle the minimal boilerplate, requiring an increase. However, I can remove it if needed.